backend/tests/routes/admin/inviteAdmin.spec.js

```diff
@@ -0,0 +1,51 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const mocha = require('mocha');
+const server = require('../../../index');
+
+const { describe } = mocha;
+const { it } = mocha;
+
+// Assertion Style
+chai.should();
+
+chai.use(chaiHttp);
+
+describe('Invite Admin', () => {
```

Comment:
Looks good.
Add a testcase which should fail. i.e for an invalid email id like `lsbvksdjbv@kdsbjv.com`
Also check what is the exact error from nodemailer and what we are responding back

Reply:
I don't think there will be any error from nodemailer if we use an invalid email. It will still send the mail.

Reply:
Also there are multiple places where instead of errStack , errorStack is written is used and therefore it doesn't work properly.
Shall I replace them?

Reply:
> I don't think there will be any error from nodemailer if we use an invalid email. It will still send the mail.

Yes, it will not really send but will not throw error too